carma-messenger-core/j2735_convertor/src/j2735_convertor_node.cpp

```diff
@@ -109,14 +109,24 @@ namespace j2735_convertor
 
 
     // Incoming geofence pub/sub
-    converted_geofence_control_pub_ = create_publisher<carma_v2x_msgs::msg::TrafficControlMessage>("incoming_geofence_control", 50);
+      // NOTE: Currently, intra-process comms must be disabled for the following two publishers that are transient_local: https://github.com/ros2/rclcpp/issues/1753
```

Comment:
Just a minor copy+paste error, we can change `following two publishers` to just `following publisher`